README.md

```diff
@@ -1,42 +1,42 @@
 [![ci](https://github.com/stlab/adobe-contract-checks/actions/workflows/ci.yml/badge.svg)](https://github.com/stlab/adobe-contract-checks/actions/workflows/ci.yml)
 [![CodeQL](https://github.com/stlab/adobe-contract-checks/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/stlab/adobe-contract-checks/actions/workflows/codeql-analysis.yml)
 
-------
+---
```

Comment:
What suddenly started rewriting markdown files?  If this was done by your personal tooling and not automated into the process the same way that clang-format is, it just amounts to source-control churn and will get out of date.  In that case, maybe the insignificant changes should be reverted.  Worst case, different peoples' personal tooling have a fight over the best way to write Markdown.

Reply:
Agree. It was unintentional.